.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+c996ed9372fc550d0a8a97b2b986bf96d26e304c
```

Comment:
```
git config blame.ignoreRevsFile .git-blame-ignore-revs
```
Can you document this somewhere? In the project's `.README`?

Reply:
You're right, I forgot about this. I added the other setup instructions to `ide_setup.md` (c.f. https://ci.apache.org/projects/flink/flink-docs-master/flinkDev/ide_setup.html) and will also add it there.

In the long run, I think we should just add all these instructions straight to `README.md`, though.